bokeh/models/tickers.py

```diff
@@ -66,7 +66,7 @@ class AdaptiveTicker(Ticker):
     The smallest allowable interval between two adjacent ticks.
     """)
 
-    max_interval = Float(100.0, help="""
+    max_interval = Float(float('Inf'), help="""
```

Comment:
So what was wrong with the previous value? There is no explanation of this change anywhere in this PR.